clean-app.sh

```diff
@@ -38,6 +43,21 @@ esac
 
 delete_data "${bundle_id}"
 
+# shellcheck disable=SC2046
+read -r -a config_bundle_ids <<< $(
+    find "${HOME}/Library/Group Containers/" \
+        -type d \
+        -maxdepth 1 \
+        -name "${config_ids}" \
+        -exec basename {} \;
+)
+for config_id in "${config_bundle_ids[@]}"; do
+    path="${HOME}/Library/Group Containers/${config_id}"
+    printf '%s' "Deleting data at ${path}... "
+    rm -r "${path}"
```

Comment:
And same here. A general rule for scripts is that they should do things forcibly unless they also provide error handling.